### ISSUE TITLE
The Park (402020)

### ISSUE THREAD

**Issue by AwesamLinux** [ValveSoftware/Proton#2564]
# Compatibility Report
- Name of the game with compatibility issues: The Park
- Steam AppID of the game: 402020

## System Information
Distro: Linux Mint 19.1 Tessa (64 bit)
GPU: Vega 56
Driver/LLVM version: Mesa 19.1.0-devel/9.0.0
Kernel version: 5.0.7-050007-generic
Proton version: 4.2-3

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms 
Game won't start, initial splash screen window shows up but then nothing happens.

## Reproduction
Try to launch the game
[steam-402020.log](https://github.com/ValveSoftware/Proton/files/3099577/steam-402020.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3099578/system_info.txt)

**Comment by kisak-valve** [comment]
Possible line of interest from the log: `fixme:process:GetNumaHighestNodeNumber (0x32fb14): semi-stub`

**Comment by AwesamLinux** [comment]
Distro: Linux Mint 19.1 Tessa (64 bit)
GPU: Vega 56
Driver/LLVM version: 19.0.1 - padoka PPA/8.0.0
Kernel version: 5.0.13-050013-lowlatency
Proton version: 4.2-7

Game is now playable, everything works out of the box except keyboard+mouse controls. When launching and starting a new game, mouse does not respond to directions unless you press down a mouse button. To fix this you can hit escape on the keyboard and then just resume the game, and everything works perfectly.

Including logs of replicating this problem:
[steam-402020_proton-4.2-7.log](https://github.com/ValveSoftware/Proton/files/3278172/steam-402020_proton-4.2-7.log)
[steam-402020_proton-4.2-7-WINE.log](https://github.com/ValveSoftware/Proton/files/3278173/steam-402020_proton-4.2-7-WINE.log)
[system_info.txt](https://github.com/ValveSoftware/Proton/files/3278174/system_info.txt)

**Comment by AwesamLinux** [comment]
- GPU: RX VEGA 56
- Driver/LLVM version: 19.2.0-devel - padoka PPA/9.0.0
- Kernel version: 5.0.13-050013-lowlatency
- Distro version: Linux Mint 19.1 Tessa (64 bit) (Cinnamon Desktop)
- Proton version: 4.2-9
- System information: [system_info.txt](https://github.com/ValveSoftware/Proton/files/3370457/system_info.txt)

Game works out of the box now without issues. Performance is a solid 60fps at 2560x1600 resolution with my setup, with just minor stuttering at the beginning of the game. 

Including a log of a complete playthrough of the game for reference (I have completed the game on Windows too and did not spot any difference).
[steam-402020.log](https://github.com/ValveSoftware/Proton/files/3370410/steam-402020.log)

Note: An oddity is that the "fullscreen" option in the game behaves the same as "fullscreen windowed", this it also does on native windows too as far as I remember. So I don't think that this is a Proton issue. Closing, because I'm not seeing any remaining issues.
